tests/core/test_exceptions_white_list_uncaught.cpp

```diff
@@ -0,0 +1,31 @@
+// Taken from https://en.cppreference.com/w/cpp/error/uncaught_exception and
+// modified
```

Comment:
This is taken from https://en.cppreference.com. Should we add Emscripten copyright here or it's fine there are files without copyright?

Reply:
I think it's fine as it is, but maybe move it to `tests/third_party` which we've recently started to move such code to.

Reply:
Done